Jenkinsfile

```diff
@@ -345,8 +345,12 @@ pipeline {
                                 // Queues up an async integration test run using default build params (master branch),
                                 // but waits up to an hour for batches of PR merges before actually running (via quietPeriod)
                                 build job: 'sync-gateway-integration', quietPeriod: 3600, wait: false
+
+                                echo 'Queueing Benchmark Run test for branch "master" ...'
+                                build job: 'sync-gateway-benchmark', parameters: [string(name: 'SG_COMMIT', value: env.SG_COMMIT)], wait: false
```

Comment:
Can we replace the existing "Benchmarks" stage with what this is doing? Doesn't belong here in the integration test stage.

Remember to copy over the master branch check from this stage.